pages/[province].js

```diff
@@ -147,6 +150,12 @@ function ProvincePage(props) {
           ) : (
             <Box w="100%" textAlign="center">
               <Spinner size="lg" />
+              {slowLoading && (
+                <Text p={4} mt={4}>
+                  Pengambilan data Provinsi {getProvinceDisplayName(province)}{' '}
+                  membutuhkan waktu lebih lama. Mohon tunggu.{' '}
```

Comment:
I think we can modify this to
"Pengambilan data untuk provinsi `<province>` membutuhkan waktu yang lebih lama. Mohon tunggu sesaat."

And please check the responsiveness too ya.

Reply:
Is this okay?
![image](https://user-images.githubusercontent.com/3627108/126591795-14e4ce4b-3bc7-48c8-9bc6-772db8307df9.png)

Reply:
Oh sorry, could you add "" in the province name?

Reply:
Sure, I will